tgui/packages/tgui/interfaces/NavigationShuttle.tsx

```diff
@@ -0,0 +1,60 @@
+import { useBackend } from '../backend';
+import { Button, Section, Stack } from '../components';
+import { Window } from '../layouts';
+
+interface DockingPort {
+  id: string;
+  name: string;
+  available: 0 | 1;
+  error: string | null;
+}
+
+interface NavigationProps {
+  destinations: DockingPort[];
+}
+
+const SHUTTLE_ALREADY_DOCKED = 'we are already docked';
+
+export const NavigationShuttle = (props, context) => {
```

Comment:
consider disabling the window with a `dimmer` while the shuttle is in movement

Reply:
I am reworking this to have different panels display information to overcome this issue

Reply:
There is a panel for destination selection, a countdown during launch, a countdown during flight and a countdown for refueling/recharge

Reply:
The buttons for door operation are also disabled during flight